studio/debugtools.py

```diff
@@ -0,0 +1,47 @@
+import __builtin__
```

Comment:
this file is not actually being used currently, but it came it handy when I was debugging file descriptor leaks, so I decided to keep it here.